llvm/lib/Target/ARM/ARMISelLowering.cpp

```diff
@@ -9508,6 +9508,42 @@ static bool isAddSubZExt(SDNode *N, SelectionDAG &DAG) {
   return false;
 }
 
+static SDValue LowerBSWAP(SDValue Op, SelectionDAG &DAG) {
```

Comment:
This is a generic expansion that doesn't belong in target code. Don't we already have this?